### PR TITLE
Configurate search _count parameter

### PR DESCRIPTION
**Pull request by hongyh13**

### CODE REVIEW

File: FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/FhirApiConstants.cs [hunk 1]
```diff
@@ -46,6 +46,6 @@ public static class FhirApiConstants
         /// Currently _count is limited to 1000 in FHIR server.
         /// https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir/overview-of-search#search-result-parameters
         /// </summary>
-        public const int PageCount = 1000;
+        public const int DefaultPageCount = 1000;
```
Comment (moria97) [inline FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/FhirApiConstants.cs:49]: Maybe we can use a new Enum FhirApiPageCount { Single = 1, Batch = 1000 }

Comment (hongyh13) [reply FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/FhirApiConstants.cs:49]: Good Suggestion!

File: FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/FhirApiDataClient.cs [hunk 2]
```diff
@@ -146,15 +146,7 @@ private Uri CreateSearchUri(BaseFhirApiOptions fhirApiOptions)
...
-            uri = uri.AddQueryString(fhirApiOptions.QueryParameters);
-
-            // add shared parameters _count
-            List<KeyValuePair<string, string>> queryParameters = new List<KeyValuePair<string, string>>
-            {
-                new KeyValuePair<string, string>(FhirApiConstants.PageCountKey, FhirApiConstants.PageCount.ToString()),
-            };
-
-            return uri.AddQueryString(queryParameters);
+            return uri.AddQueryString(fhirApiOptions.QueryParameters);
```
Comment (sowu880) [inline FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/FhirApiDataClient.cs:149]: Could remove useless usings "System.Collections.Generic;"

Comment (hongyh13) [reply FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/FhirApiDataClient.cs:149]: Sure, removed.

**Review by moria97 — APPROVED** [review]

**Pull request merged**